examples/image/imageSize.rb

```diff
@@ -0,0 +1,4 @@
+Shoes.app do
+    size "http://shoesrb.com/manual/static/shoes-icon.png"
```

Comment:
Similar with what Noah says I don't see this as part of the Shoes DSL ([I checked the manual here](https://github.com/scarpe-team/scarpe/blob/main/docs/static/manual.md)). 

but open if you can point me to places where size is used like this